commcare_connect/templates/opportunity/add_visits_existing_users.html

```
@@ -105,192 +102,184 @@
-      <div class="flex relative mx-auto items-center justify-between w-full px-4 bg-slate-50 rounded-lg shadow-sm h-14">
-        <ul class="tabs">
-          {% for tab in tabs %}
-          <li
-            @click="setTab('{{ tab.key }}')"
-            id="{{ tab.key }}-tab"
-            :class="{ 'active': selectedTab === '{{ tab.key }}' }"
-          >
+    </script>
+  {% endblock extra_js %}
+  {# djlint:off — H025: djlint flags the root content div as an orphan in {% extends %} templates; it is properly closed within this file #}
+  <div class="md:max-w-screen-lg mx-auto flex flex-col gap-6"
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)